pyspectral/tests/test_rayleigh.py

```diff
@@ -229,16 +306,16 @@ def test_get_reflectance_dask(self, get_effective_wvl,
         sun_zenith = da.from_array(np.array([67., 32.]), chunks=2)
         sat_zenith = da.from_array(np.array([45., 18.]), chunks=2)
         azidiff = da.from_array(np.array([150., 110.]), chunks=2)
-        blueband = da.from_array(np.array([14., 5.]), chunks=2)
-        retv = self.viirs_rayleigh.get_reflectance(sun_zenith, sat_zenith, azidiff, 'M2', blueband)
+        redband_refl = da.from_array(np.array([14., 5.]), chunks=2)
```

Comment:
Any reason why not using `da.array()` directly? Same is used many places, so would simplify the code a lot.

Reply:
No idea actually, don't think I did those. But I can change,you are right. Can you actually set the `chunksize` that way? Not that it should matter in this case though...